addons/popochiu/engine/objects/graphic_interface/templates/sierra/sierra_gi.gd

```diff
@@ -0,0 +1,117 @@
+extends PopochiuGraphicInterface
+
+
+# ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ GODOT ░░░░
+func _ready() -> void:
+	super()
+	
+	Cursor.replace_frames($Cursor)
+	Cursor.show_cursor()
+	
+	$Cursor.hide()
+	
+	$Menu.hide()
+	
+	E.current_command = SierraCommands.Commands.WALK
+	
+	%SierraSettingsPopup.option_selected.connect(_on_settings_option_selected)
+
+
+func _input(event: InputEvent) -> void:
+	if D.current_dialog: return
```

Comment:
You should be explicitly blocking when the dialog is running and relying on the blocking status.

Reply:
Done, but needs more testing because while I was making the change I remembered something about having problems in specific scenarios.